libibverbs/verbs.h

```diff
@@ -1076,6 +1078,8 @@ enum ibv_wr_opcode {
 	IBV_WR_BIND_MW,
 	IBV_WR_SEND_WITH_INV,
 	IBV_WR_TSO,
+	IBV_WR_MASKED_ATOMIC_CMP_AND_SWP,
+	IBV_WR_MASKED_ATOMIC_FETCH_AND_ADD,
```

Comment:
Changing the value of IBV_WR_DRIVER1 is ABI breaking

Reply:
Thank you, I'll do more work on this.